salt/modules/opsgenie.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+Module for sending data to OpsGenie
+
+.. versionadded:: 2017.7.2
+
+:configuration: This module can be used in Reactor System for
+    posting data to OpsGenie as a remote-execution function.
+    For example:
+    .. code-block:: yaml
+        ﻿opsgenie_event_poster:
+          local.opsgenie.post_data:
+            - tgt: 'salt-minion'
+            - kwarg:
+                name: event.reactor
+                api_key: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+                reason: {{ data['data']['reason'] }}
+                action_type: Create
+"""
+
+import salt.exceptions
+import requests
+import json
+import logging
+
+API_ENDPOINT = "https://api.opsgenie.com/v1/json/saltstack?apiKey="
+
+log = logging.getLogger(__name__)
+
+def post_data(api_key=None, name='OpsGenie Execution Module', reason=None, action_type=None):
+    if api_key is None or reason is None or action_type is None:
```

Comment:
Can you add some documentation here?

This should include:
- Description of the function
- Description of each argument
- CLI Example of how to use the function

Please see other execution modules for an example.